### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by RICKCOYL**
### In this iteration, we

- Created a directory structure for our bookstore app
- Created Components: BooksList, BookForm, Book, and App
- Displayed a list of books in our BooksList 
- Composed the Components into an 
- created basic actions and reducers
- Created a state with redux for our app

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div>
+      <table>
+        <tbody>
+          <tr>
+            <th>BookId</th>
+            <th>title</th>
+            <th>category</th>
+          </tr>
+        </tbody>
```
Comment (BrittanyBlake) [inline src/containers/BooksList.js:14]: - [ ] Your BookList component should display a list of the books using the `Book` component
Requirement from pathwright 👇🏾 
Screen Shot 2021-04-25 at 12 27 24 PM

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import Book from '../components/Book';
+
+export default function BooksList() {
```
Comment (BrittanyBlake) [inline src/containers/BooksList.js:4]: - [ ] The `book object` should be passed as a prop and passed to each `Book` component

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @RICKCOYL and @Morrism1! 👋 
 You have also done a wonderful job with this milestone so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
**BooksList Component**

- [ ]  The booklist component should be connected to the redux store using the [connect() function](https://react-redux.js.org/api/connect) or [Redux hooks](https://react-redux.js.org/api/hooks) so that it can fetch the books from the state. 
Requirements from pathwright 👇🏾 
Screen Shot 2021-04-25 at 12 25 19 PM

- [ ] Please make sure you are passing the `book` object as a prop. You should be rendering the `Book` component in the body of the table and passing the `book` object as a prop to each `Book` component.

****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**## Hi @lap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**